src/qlever/commands/ui.py

```diff
@@ -59,57 +102,133 @@ def execute(self, args) -> bool:
                 log.info("")
 
         # Construct commands and show them.
-        server_url = f"http://{args.host_name}:{args.port}"
+        pull_latest_image = "/" in args.ui_image and not args.no_pull_latest
+        ui_config_name = args.name
```

Comment:
suggestion: use `UI_CONFIG` from the `ui` section

Reply:
I was undecided whether we even need the `UI_CONFIG` anymore with the new setup

Reply:
Removing `UI_CONFIG` would also be fine.